packages/flame_test/example/pubspec.yaml

```diff
@@ -12,7 +12,8 @@ dependencies:
     sdk: flutter
   flame: ^1.1.0
 dev_dependencies:
-  flame_test: ^1.2.0
+  flame_test:
+    path: ../
```

Comment:
Hmm I think you need to run a `melos clean` right?

Reply:
I think that melos don't handle examples :/